#publisher type defines messages for person tracking node(RecognRsPersonTracker or Frame message)
# supported values:
#     defaultPublisher - default message format - main message - "Frame"
publisherType: "defaultPublisher"

#test mode, true - publish person tracking topic with color image
isTestMode: true

pointingGestureEnabled: false
waveGestureEnabled: false
skeletonEnabled: false
recognitionEnabled: false
trackingEnabled: true
headPoseEnabled: false

#load Data saved
loadDb: false
dbPath: '/home/elisa/testPT1.db'

headPoseEnabled: false
headBoundingBoxEnabled: false
landmarksEnabled: false

